.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 .env
 pgdata/
+/backend/postgres-data/
```

Comment:
`pgdata/` and `/backend/postgres-data/` same purpose?

Reply:
@aidan-lane what is `pgdata/` for?

Reply:
pgdata was a remnant from testing I think, if the volume is called postgres-data then we can remove pgdata